examples/todo-list/src/models/todo-list.model.ts

```diff
@@ -30,7 +31,7 @@ export class TodoList extends Entity {
   color?: string;
 
   @hasMany(() => Todo)
-  todos?: Todo[];
+  todos: Todo[];
```

Comment:
If the property is NOT optional, when creating a new todo list, you have to specify the navigational property which IMO is something we want to prevent...

Reply:
If you try to create a `todo-list` with a `todos` property, it'll get rejected because the `todos` property will be considered an additional property. Properties under the `@hasMany` and `@hasOne` decorators aren't treated the same as regular properties. This change was so that it matches what's outputted by `lb4 relation`.

Reply:
> Properties under the @hasMany and @hasOne decorators aren't treated the same as regular properties.

I wasn't aware of that, thank you for the explanation!